cabal.project

```diff
@@ -1,4 +1,6 @@
-packages: 
-  ./jl4
+packages:
+  ./jl4-core
+  ./jl4-lsp
+  ./jl4-cli
```

Comment:
I think renaming all the example paths is bad for discoverability. Perhaps we should keep `jl4-cli` as `jl4`, then the examples would not have to be moved?